config/routes.rb

```diff
@@ -1,10 +1,21 @@
-Rails.application.routes.draw do
+# frozen_string_literal: true
+
+Rails.application.routes.draw do   
+  devise_for :model_1s
```

Comment:
- [ ] This should be `devise_for: users` because devise configurations are in the `user` model

Reply:
okay thanks